src/org-units-selector/OrgUnitsSelector.jsx

```diff
@@ -141,17 +113,60 @@ export default class OrgUnitsSelector extends React.Component {
                 children: true,
             },
             ...listParams,
-            ..._.omit(options, ["postFilter"]),
         };
 
-        const response = api.models.organisationUnits.get(listOptions);
+        if (search) {
+            return api.models.organisationUnits.get({
+                ...baseOptions,
+                paging: true,
+                pageSize: 1000,
+                filter: { displayName: { ilike: search } },
+            });
+        } else if (rootIds) {
+            let cancel = false;
```

Comment:
Using `cancel` in the closure, cool! This {getData, cancel} object is not exactly a full D2ApiResponse, but as it's all it's used in the code, is ok.

Reply:
Yeah, we should look at how to do this in the future and standardize it in all the apps.